app/common/ValueFormatter.ts

```diff
@@ -79,19 +79,25 @@ export class BaseFormatter {
    * Formats using this.format() if a value is of the right type for this formatter, or using
    * AnyFormatter otherwise. This method the recommended API. There is no need to override it.
    */
-  public formatAny(value: any): string {
-    return this.isRightType(value) ? this.format(value) : formatUnknown(value);
+  public formatAny(value: any, translate?: Function): string {
+    return this.isRightType(value) ? this.format(value, translate) : formatUnknown(value);
   }
 
   /**
    * Formats a value that matches the type of this formatter. This should be overridden by derived
    * classes to handle values in formatter-specific ways.
    */
-  protected format(value: any): string {
+  protected format(value: any, translate?: Function): string {
```

Comment:
```suggestion
  protected format(value: any, _translate?: (val: string) => string): string {
```

Sorry for nitpicking. This gets flagged as unused in some editors without the prefix.